tensorflow/compiler/mlir/tensorflow/ir/tf_ops_a_m.cc

```diff
@@ -1246,6 +1241,139 @@ LogicalResult Conv2DOp::UpdateDataFormat(StringRef data_format) {
   return success();
 }
 
+LogicalResult Conv2DOp::inferReturnTypes(
+    mlir::MLIRContext* context, llvm::Optional<mlir::Location> location,
+    mlir::ValueRange operands, mlir::DictionaryAttr attributes,
+    mlir::RegionRange regions,
+    llvm::SmallVectorImpl<mlir::Type>& inferredReturnTypes) {
+  const int64_t num_spatial_dims = 2;
```

Comment:
Use the auto-generated op adaptor verify method to avoid manual implementation of some of the checks.

For example,
https://github.com/tensorflow/tensorflow/blob/61dcdcb2fc0869c9ee3486aadbe0c8cbb723cf82/tensorflow/compiler/mlir/lite/ir/tfl_ops.cc#L1453

Reply:
Done. Thanks.